_posts/2023-10-10-endless-os.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: distro
+author: "Sam Hewitt"
+updater: "cassidyjames"
+title: "Endless OS"
+date: "2013-09-14"
```

Comment:
Why did you change this from the previous value of 2018-06-01?

Reply:
That's an excellent question; I meant `2023-09-14`, as that was the latest release 😄 

```suggestion
date: "2023-09-14"
```

but I think I misunderstood the directions in the PR template

Reply:
Ah, yes, those instructions (and CONTRIBUTING.md too) could be clearer. This date is supposed to be the original date the distro was added to the site. It may help to think of these metadata fields as being related to the post, not the distro. I will change those files to make this more obvious.

We could add a new `latest_release` subfield under `notes` (since those are indeed related to the distro itself), but I'm afraid that would require much more frequent updates to remain correct, so it's probably best to avoid such time-sensitive data.

As for this PR, yes, let's restore the `2018-06-01` date.